src/main/java/racingcar/Application.java

```diff
@@ -1,7 +1,25 @@
 package racingcar;
 
+import java.util.List;
+import racingcar.util.GameSetter;
+import racingcar.util.NumberSetter;
+import racingcar.view.InputView;
+import racingcar.view.OutputView;
+
 public class Application {
     public static void main(String[] args) {
         // TODO 구현 진행
+        List<Car> cars = GameSetter.getCars(InputView.getCarNames());
+        Race race = new Race(cars);
+        int carNumber = cars.size();
+        int attempt = InputView.getAttemptNumber();
+
+        OutputView.printResultTitle();
+        while (attempt > 0) {
+            TurnResult result = race.getResultForEachTurn(NumberSetter.getRandomNumbers(carNumber));
+            OutputView.printEachTurnResult(result.getNames(), result.getPositions());
+            attempt--;
+        }
```

Comment:
while 문보다는 실수를 덜 할 수 있는 For문은 어떨까요?
```suggestion
        for (int i = 0; i < attempt; i++) {
            TurnResult result = race.getResultForEachTurn(NumberSetter.getRandomNumbers(carNumber));
            OutputView.printEachTurnResult(result.getNames(), result.getPositions());
        }
```

Reply:
반복문 탈출조건이 명확한 for 문을 쓰는게 확실히 실수 방지에 좋을 것 같네요. 감사합니다.